readme.md

```diff
@@ -33,7 +33,11 @@ The first time you click on the extension icon, it will ask you for access to br
 
 #### Notifications permission
 
-If you want to receive desktop notifications, you can enable them on extension options page. You will then be asked for the notifications permission.
+If you want to receive desktop notifications for public repositories, you can enable them on extension options page. You will then be asked for the notifications permission.
+
+#### Repos permission
+
+If you want to receive (useful) desktop notifications for any private repositories you have, you will be asked for the repo permission as well. This is due to GitHub's current permission scheme, as the only way we can read anything about your private repos is if we have full control over them. If you're concerned with your security in this manner please feel free to not give this permission, just be aware that if you do not enable this permission, clicking on the notification will take you to the new tab page since we can't get any information about the repo you got the notification for.
```

Comment:
Can we make it so clicking on the notification would take user to `/notifications` page for private repos?